llvm/include/llvm/MC/MCDisassembler/MCDisassembler.h

```diff
@@ -131,6 +131,11 @@ class MCDisassembler {
   ///                   MCDisassembler::SoftFail if the instruction was
   ///                                            disassemblable but invalid,
   ///                   MCDisassembler::Fail if the instruction was invalid.
+  ///
+  /// Note: to avoid potential issues caused by the field
```

Comment:
The majority of the targets don't do this.

Reply:
One of our internal builds (sanitizers) fails if we do not set the CommentStream for the target:

```
<...>/llvm-project/llvm/lib/MC/MCDisassembler/MCDisassembler.cpp:33:55: runtime error: reference binding to null pointer of type 'raw_ostream'
SUMMARY: UndefinedBehaviorSanitizer: undefined-behavior <...>/llvm-project/llvm/lib/MC/MCDisassembler/MCDisassembler.cpp:33:55 
```

Maybe I can just remove the comment?